PaolaSantolloV/react-apprenticeship-capstone4
# Feat/deliverable3

## Description
This PR contain the 3 deliverable:  Fetching a Real API, React Router, Global State Management deliverable

- Add global state
- Add the apis on the project
- Add router
- Add paggination functionality

## Type of Change

```
Delete options that are not relevant.
```


## How should this be tested?
In your terminal runnpm installafter that` npm run start,` now you can see the page on[ localhost:3000](https://localhost:3000/)
To run the test` npm run test`, if you want to watch the coverage `npm run test a -- --coverage`.



## What is the deliverable?
deliverable 3

## Screenshots and links
netlify link : [https://vigilant-euclid-5c9c68.netlify.app/home](https://vigilant-euclid-5c9c68.netlify.app/home)


## Checklist
